### PR TITLE
D8CORE-4551: updated permissions for contributors and site editors

### PR DESCRIPTION
**Pull request by imonroe**
# READY FOR REVIEW

# Summary
- We are altering some permissions for contributors and site editors.  We are adding the site-embedder role.

# Review By (Date)
- code freeze.

# Criticality
- 2

# Urgency
- Normal

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Rebuild Cache and import config `drush cr ; drush ci`
3. Check to make sure the permissions are set correctly.

### Site Configuration Sync

- Is there a config:export in this PR that changes the config sync directory? YES

## Front End Validation
- [ ] Design is approved by @ user?
- [ ] HTML validation: Is the markup using the appropriate semantic tags and [passes validation](https://validator.w3.org/nu/)? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Cross-browser testing: Has been performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Automated accessibility: Scans performed? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?
- [ ] Manual accessibility: Manually tested? Or, [QA request ticket created](https://github.com/SU-SWS/template_warehouse/blob/master/jira_templates/QA_request_template.txt)?

### Code
- [ ] Are the naming conventions following our standards?
- [ ] Does the code have sufficient inline comments?
- [ ] Is there anything in this code that would be hidden or hard to discover through the UI?
- [ ] Are there any [code smells](https://blog.codinghorror.com/code-smells/)?
- [ ] Are tests provided? eg (unit, behat, or codeception)

### Code security
- [ ] Are all [forms properly sanitized](https://www.drupal.org/docs/8/security/drupal-8-sanitizing-output)?
- [ ] Any obvious [security flaws or new areas for attack](https://www.drupal.org/docs/8/security)?

## General
- [ ] Is there anything included in this PR that is not related to the problem it is trying to solve?
- [ ] Is the approach to the problem appropriate?

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket(s)
- Other PRs
- Any other contextual information that might be helpful (e.g., description of a bug that this PR fixes, new functionality that it adds, etc.)
- Anyone who should be notified? (`@mention` them here)

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

**Comment by imonroe** [comment]
@buttonwillowsix -- Would you please have a look at this and let me know if I have correctly captured all the changes?  Between the ticket and my notes, it got a little confusing....

File: config/sync/user.role.contributor.yml [hunk 1]
```diff
@@ -49,21 +49,16 @@ permissions:
...
   - 'create image media'
   - 'create media'
   - 'create policy log'
-  - 'create stanford_course content'
   - 'create stanford_event content'
-  - 'create stanford_event_series content'
   - 'create stanford_gallery_images media'
   - 'create stanford_news content'
   - 'create stanford_page content'
-  - 'create stanford_person content'
```
Comment (buttonwillowsix) [inline config/sync/user.role.contributor.yml:65]: I think we may want to leave this. Large site owners, it turns out, sometimes use student workers to add alumni profiles, etc.

Comment (imonroe) [reply config/sync/user.role.contributor.yml:65]: Fixed.

File: config/sync/user.role.su_site_embedder.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+uuid: 9dedbe8e-27b6-40de-95fd-29e4efe67be4
+langcode: en
+status: true
+dependencies:
+  module:
+    - field_permissions
+id: su_site_embedder
```
Comment (pookmish) [inline config/sync/user.role.su_site_embedder.yml:7]: can you add a couple codeception tests for this role

**Review by pookmish — CHANGES_REQUESTED** [review]
Just looking for a little test

File: config/sync/user.role.su_site_embedder.yml (hunk 2 above)
Comment (imonroe) [reply config/sync/user.role.su_site_embedder.yml:7]: can do.

**Review by pookmish — APPROVED** [review]

**Pull request merged**